src/common/query-parameters.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { IsNumber, IsOptional, IsString } from 'class-validator';
+
+export class QueryParameters {
```

Comment:
this class should be more "precisely" named. Imagine that you could have more queries to validate. it could be something like `FindAllRecipesQuery`